docs/next/guides/building-and-testing/building.md

```diff
@@ -42,7 +42,7 @@ During this process, the source files located in the `src/*` directory are trans
 * `./dist/handsontable.full.min.css`
 * `./dist/languages/*`
 
-More info about dist packages can be found [here](https://github.com/handsontable/handsontable/blob/master/dist/README.md.
+More info about dist packages can be found [here](https://github.com/handsontable/handsontable/blob/master/dist/README.md).
```

Comment:
Can we move this change to the separated PR (as grouped "documentation polishing" changes)?